packages/babel-plugin-transform-classes/src/index.js

```diff
@@ -21,6 +21,8 @@ export default declare((api, options) => {
 
   const { loose } = options;
 
+  const setClassMethods = options.loose || api.assumption("setClassMethods");
```

Comment:
Which option should have the precedence? `loose` is set directly in the plugin, but `assumption` is the new modern option.

Reply:
I think `loose` should be prioritized. We can use `api.assumption("setClassMethods");` as default value for `loose`. So `loose: false` can not be overridden by `api.assumption("setClassMethods");`.

Reply:
Do we want some kind of warning somewhere if both are specified?